ch4/arm_ch4p8.md

```diff
@@ -529,7 +529,7 @@ Now that we have a good understanding of the limits of our regression model, we
 * `intercept`: this represents the average score for a course taught by a male native English speaking professor of average score, which award more than one credit. Classes which corresponds to this description are usually scored 3.69 points 
 * `female`: this coefficient represents the difference in intercept between courses taught by female and male professors. The negative coefficient means that on average, every else given, classes taught by female professors are more likely to be scored less. Fortunately, such kind of discimination doesn't seem to be neither extremely statistically significant nor having any remarkable effect size
 * `onecredit`: interestingly, anything else being equal, one credit courses seems to be scored higher. This could make intuitive sense, since those are generally easier than 6 or 12 credit courses. More interestingly is that the coefficient, which is statistically significant, is also quite large compared to the other predictors, which means this is generally an important factor when describing course evaluations
-* `profevaluation`: 0.94 represents the slope of the line for a course taught by a male native English speaking professor, which is worth more than one credit. A one standard deviation increase increase in the score given to the professor corresponds on +0.94 increase in class score
```

Comment:
I think this is actually correct. If you notice, we are dividing by 2 standard deviations instead of 1. The reason why this is done is explained in this same chapter if I don't remember wrongly ― basically, this is needed to make the coefficients between binary variables and continuous variables comparable.

Reply:
I agree the division by 2 SDs is to make binary and continuous variables comparable. A binary variable that takes the value 0 half of the time and 1 the other half of the time has an SD of 0.5, so going from 0 to 1 is 2 SDs. That is why Gelman proposes scaling continuous variables by 2 SDs - so their coefficients represent a comparable increase to that of a binary variable going from 0 to 1 [i.e., "off" to "on"].